h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -99,6 +102,35 @@ function TextField({
 }
 
 export default function CreateGroupForm() {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  const createGroup = async () => {
```

Comment:
I think you can wrap this in [`useCallback`](https://preactjs.com/guide/v10/hooks/#usecallback), to avoid a new function reference to be created on every render of this component, which would in turn re-render the `Button` where this is passed as `onClick`.